### PR TITLE
Added aiohttp-graphql

### PR DESCRIPTION
**Pull request by dfee**
Ref: https://github.com/aio-libs/aiohttp/issues/2288

### CODE REVIEW

File: docs/third_party.rst [hunk 1]
```diff
@@ -56,6 +56,9 @@ aiohttp extensions
 
 - `aiohttp-jinja2 <https://github.com/aio-libs/aiohttp-jinja2>`_ Jinja2
   template renderer for aiohttp.web.
+  
+- `aiohttp-grapqhql <https://github.com/graphql-python/aiohttp-graphql>`_
```
Comment (socketpair) [inline docs/third_party.rst]: Typo in project name :)

Comment (kxepal) [reply docs/third_party.rst]: Nice found, @socketpair !

Comment (dfee) [reply docs/third_party.rst]: It nicely mirrors how often I mistype `graphql` in vim. Thanks for finding!

**Comment by socketpair** [comment]
@dfee please ammend you current commit, and do `push -f`. Also, please rebase on current master branch.

**Comment by dfee** [comment]
Changes submitted. Also, this was a pretty good tutorial for how to rebase against `aio-libs/aiohttp`: https://robots.thoughtbot.com/keeping-a-github-fork-updated

I always forget how to do this.

**Comment by socketpair** [comment]
Please squash commits into one.

**Comment by dfee** [comment]
I believe we're good to go now. Let me know!

**Comment by asvetlov** [comment]
The library is not in `aio-libs` team, the record should be moved down to http://aiohttp.readthedocs.io/en/stable/third_party.html#others

**Comment by dfee** [comment]
mind if i lexicographically sort that list while I'm at it?

**Comment by asvetlov** [comment]
I just added to the end but sorting makes sense.

**Comment by asvetlov** [comment]
Pushed the link into proper place.

@dfee your library is young. I wish all the bests but right now it doesn't fit to *highly recommended* list.

Feel free to make a PR for sorting lists lexicographically.

**Comment by asvetlov** [comment]
@socketpair please merge PR when green.

**Pull request merged**